addons/common/XEH_postInit.sqf

```diff
@@ -55,6 +56,24 @@
     _vehicle setVariable [QGVAR(blockEngine), _set > 0, true];
     _vehicle engineOn false;
 }] call CBA_fnc_addEventHandler;
+[QGVAR(blockThrow), {
+    params ["_object", "_set"];
+    TRACE_2("blockThrow EH",_object,_set);
+    if (_set > 0) then {
+        GVAR(blockThrow_EH) = ["KeyDown", {(_this select 1) in actionKeys "Throw"}] call CBA_fnc_addDisplayHandler;
```

Comment:
Due to what @jonpas said about joystick and how `actionKeys` handles keys with modifiers, it looks like I have to change vanilla throw blocking way. Maybe the best is not to block vanilla throw but advanced throw only. Will think some time about it.